docs/v0.2.0/dev/DOCS.md

```diff
@@ -0,0 +1,63 @@
+# Working on the Envoy Gateway Docs
+
+The documentation for the Envoy Gateway lives in the `docs/` directory. Any
+individual document can be written using either [reStructuredText] or [Markdown],
+you can choose the format that you're most comfortable with when working on the
+documentation.
+
+## Documentation Structure
+
+We supported the versioned Docs now, the directory name under docs represents
+the version of docs. The root of the latest site is in `docs/latest/index.rst`.
+This is probably where to start if you're trying to understand how things fit together.
+
+Note that the new contents should be added to `docs/latest` and will be cut off at
+the next release. The contents under `docs/v0.2.0` are auto-generated,
+and usually do not need to make changes to them, unless if you find the current release pages have
+some incorrect contents. If so, you should send a PR to update contents both of `docs/latest`
+and `docs/v0.2.0`.
+
+It's important to note that a given document _must_ have a reference in some
+`.. toctree::` section for the document to be reachable. Not everything needs
+to be in `docs/index.rst`'s `toctree` though.
+
+You can access the website which represents the current release in default,
+and you can access the website which contains the latest version changes in
+[Here][latest-website] or at the footer of the pages.
+
+## Documentation Workflow
+
+To work with the docs, just edit reStructuredText or Markdown files in `docs`,
+then run
+
+```bash
+make docs
+```
+
+This will create `docs/html` with the built HTML pages. You can view the docs
+either simply by pointing a web browser at the `file://` path to your
+`docs/html/latest`, or by firing up a static webserver from that directory, e.g.
+
+``` shell
+cd docs/html/latest ; python3 -m http.server
```

Comment:
This should be `docs/html` to support versioned docs. Otherwise, `code 404, message File not found` when trying to access a versioned doc.

Reply:
Thanks, I forgot to update this, Will be updated later.

Reply:
<img width="1068" alt="image" src="https://user-images.githubusercontent.com/48784001/199169321-1eb27fa6-a2ee-4276-ba97-e3ec0be362d3.png">

Test locally and works fine.